### ISSUE TITLE
2 classes or multi classes segmentation

### ISSUE THREAD

**Issue by Kalfredwv** [bhpfelix/segment-anything-finetuner#5]
Thank you for your help, but does it aim at 2 classes or multi classes segmentation?

**Comment by zhengziqiang** [comment]
No. SAM does not support the close-domain semantic segmentation. You need to combine the semantics through the post-processing or use the semantic prompts to generate the masks.
